mr/src/main/java/org/apache/iceberg/mr/mapred/IcebergSerDe.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iceberg.mr.mapred;
```

Comment:
can we put the hive classes in org.apache.iceberg.hive ? This is committed, but are you guys ok for this refactor?

Reply:
I thought the convention was that the package name needed to match the subproject name and this isn't in the `hive` package but maybe that's not the case? Alternatively they could go in `org.apache.iceberg.mr.hive`?

Reply:
I prefer `org.apache.iceberg.mr.hive`

Reply:
`mr.hive` sounds good to me.

I don't think we need to worry too much about this kind of refactor right now. We expect it to change rapidly as we build. We'll include a note in any release about how it is experimental and subject to change.

Reply:
OK, my preference would be to leave this as it is for now and then do a review of all the packaging once we have the StorageHandler and InputFormat merged.